templates/careers/admin.html

```diff
@@ -1,24 +1,87 @@
+    </div>
+  </div>
+  <div class="p-strip is-shallow">
+    <div class="u-fixed-width u-hide--small u-hide--medium">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/fcdf52cb-department-admin-%402x.jpg",
+        alt="",
+        width="2216",
+        height="834",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+  </div>
+  
+  <div class="row">
+    <div class="col-6">
+      <h3>Admin at Canonical</h3>
+    </div>
+    <div class="col-6">
+      <p>We are a multi-skilled and multi-cultural team. We encourage new members of the team to develop skills across the entire range of admin challenges so that we can help one another as needed and bring our joint insights to bear on continuous improvement. Our team is largely based in offices in Austin, Boston, London, Bratislava, Beijing, Taipei and Tokyo, but we offer some flexibility to work from home. We often run company events around the world in person, so you need the latitude to be in a completely different timezone for up to two weeks at a time, three or four times a year.</p>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <h2>See for yourself the work we're doing</h2>
+    <ul class="p-inline-list">
+      <li class="p-inline-list__item u-no-margin--right">
+        <a class="github-button" href="https://github.com/canonical" data-size="large" aria-label="Follow @canonical on GitHub"></a>
+      </li>
+      <li class="p-inline-list__item u-no-margin--right">
+        <a class="github-button" href="https://github.com/canonical/canonical.com" data-icon="octicon-star" data-size="large" data-show-count="true" aria-label="Star canonical/canonical.com on GitHub">Star</a>
+      </li>
+      <li class="p-inline-list__item">
+        <a class="github-button" href="https://github.com/canonical/canonical.com/fork" data-icon="octicon-repo-forked" data-size="large" data-show-count="true" aria-label="Fork canonical/canonical.com on GitHub"></a>
+      </li>
+      <li class="p-inline-list__item">
+      <a href="https://www.linkedin.com/company/234280" class="p-icon--linkedin">LinkedIn</a>
+    </li>
+    <li class="p-inline-list__item">
+      <a href="https://twitter.com/Canonical" class="p-icon--twitter">Twitter</a>
+    </li>
+    <li class="p-inline-list__item">
+      <a href="https://www.youtube.com/user/celebrateubuntu" class="p-icon--youtube">YouTube</a>
+    </li>
+    </ul>
+  </div>
+</section>
+
+{% with section_classes='p-strip', heading_topic='Admin', tag_id='1216', limit='3' %}
+  {% include "shared/_blog-card.html" %}
+{% endwith %}
+
```

Comment:
This could also use the Python [blog module](https://github.com/canonical/canonicalwebteam.blog#readme)